app/mobile-content/pages/shared/page-input/page-input.component.html

```diff
@@ -0,0 +1,161 @@
+<div class="page-form">
```

Comment:
Hm... I wonder if it would be possible to get rid of this bunch of conditional elements. Can there be maybe multiple of these page-input-component templates and depending on the page type, a different one gets used? It's not important, but I feel it would be cleaner.